execution/engine/engine.go

```diff
@@ -89,12 +89,8 @@ func (ee *Engine) NotifyForkchoiceUpdate(
 	switch {
 	// We do not bubble the error up, since we want to handle it
 	// in the same way as the other cases.
-	case errors.IsAny(
-		err,
-		engineerrors.ErrAcceptedPayloadStatus,
-		engineerrors.ErrSyncingPayloadStatus,
```

Comment:
The only reason I could imagine for putting these to together is if some EL could return them at the same time?

Reply:
All good